locales/en/public.json

```diff
@@ -64,21 +64,21 @@
   "AutomatedAnalysisConfigDrawer": {
     "INPUT_GROUP": {
       "CREATE_RECORDING": {
-        "LABEL": "Create Recording"
+        "LABEL": "Create recording"
       },
       "OPEN_SETTINGS": {
         "LABEL": "Open Settings"
       }
     }
   },
   "AutomatedAnalysisConfigForm": {
-    "CURRENT_CONFIG": "Current Configuration",
-    "FORM_TITLE": "Profiling Recording Configuration",
+    "CURRENT_CONFIG": "Current configuration",
+    "FORM_TITLE": "Profiling recording Configuration",
     "FORMATTED_TEMPLATE": "Name: {{template.name}}, Type: {{template.type}}",
     "MAXIMUM_AGE": "Maximum age ({{unit}})",
     "MAXIMUM_SIZE": "Maximum size ({{unit}})",
     "SAVE_CHANGES": "Save changes",
-    "TEMPLATE_HELPER_TEXT": "The Event Template to be applied to Automated Analysis recordings.",
+    "TEMPLATE_HELPER_TEXT": "The Event template to be applied to Automated Analysis recordings.",
```

Comment:
https://www.patternfly.org/ux-writing/capitalization/#capitalization-guidelines

I think according to these guidelines, we can consider an `Event Template` to be a "custom resource" and a proper noun - it can stay as `Event Template`. Same would apply to `Automated Rule`, but maybe not to `automated analysis` (since analyses aren't a resource that is created).